style.css

```diff
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  background-color: #fff;
+}
+
+header,
+.main-content-container {
+  padding: 0 24px;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+#logo,
+.fa-bars {
+  color: #6070ff;
+}
+
+#logo {
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+/* main section */
+.main-content {
+  height: 500px;
+  display: flex;
+  align-items: center;
+  background-image: url("asset/header-bg.png");
```

Comment:
- [ ] ***[OPTIONAL]*** The background image you have used in this section is nearly perfect. Kindly fix the design to match the given template. I think fixing the position/size of the background image would work.
  - ***Given Design***
  ![image](https://user-images.githubusercontent.com/60083437/219165623-4ea0da85-5dcd-4f47-8215-873801d2cccb.png)
  - ***Your Design***
  ![image](https://user-images.githubusercontent.com/60083437/219167460-6bf82b32-2edd-4db0-9aed-8fe27bc4572f.png)